src/rules/color-no-named/__tests__/index.js

```diff
@@ -0,0 +1,53 @@
+import {
+  ruleTester,
+  warningFreeBasics,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule(undefined, tr => {
+  warningFreeBasics(tr)
+
+  tr.ok("a { color: #000; }")
+  tr.ok("a { color: #ababab; }")
+  tr.ok("a { color: rgba(0, 0, 0, 0); }")
+  tr.ok("a { something: #000, #fff, #333; }")
+
+  tr.ok("a { padding: 000; }")
+  tr.ok("a::before { content: \"#ababa\"; }")
+
+  tr.ok("a { border-#$side: 0; }", "ignore sass-like interpolation")
+  tr.ok("a { box-sizing: #$type-box; }", "ignore sass-like interpolation")
+  tr.ok("a { color: $black; }", "ignore sass-like interpolation")
```

Comment:
Sorry. I misunderstood the meanings of `interpolation`. As you say, just variable name.